docs/guides/integration/pytorch.md

```diff
@@ -0,0 +1,115 @@
+# Installing pytorch with uv
+
+[PyTorch](https://pytorch.org/) is a popular open-source deep-learning learning library that can run
+both on CPUs and GPUs. For this reason, PyTorch can be notoriously cumbersome to install, especially
+on Linux/Windows.
+
+[PyTorch website](https://pytorch.org/get-started/locally/) offers a simple tool to determine what
+`pip` command you should run to install PyTorch. This guide should help you do the same with `uv`.
+If the following instructions fail, you might want to check the original source for any difference,
+and open an issue to report the discrepancy. In this case, or if you run into any other issue,
+please refer to [Getting Help](../../getting-started/help.md).
+
+!!! tip "TL;DR"
+
+    * If want PyTorch on macOS, just run `uv add torch`/`uv pip install torch`.
+    * If want PyTorch on Windows *with CPU support*, just run `uv add torch`/`uv pip install torch`.
+    * If want to install PyTorch on Linux with CPU support, add `--extra-index-url=https://download.pytorch.org/whl/cpu` to the `uv add`/`uv pip install` command.
```

Comment:
Would we be able to do this with `--index-url` instead of `--extra-index-url` ? 
The extra version of this flag was the cause of a bad security event already for PyTorch and we would not want to repeat that here: https://pytorch.org/blog/compromised-nightly-dependency/

Because we do use `--index-url` for all the pip commands, you can rely on the fact that the url will contain all depedencies needed to install torch.

Reply:
Note we have different behavior than pip to try to avoid such issues https://docs.astral.sh/uv/pip/compatibility/#packages-that-exist-on-multiple-indexes

I think the real answer is that we need #171 though

Reply:
Uh, he has a point. I'll add a link to explain this behaviour.

Reply:
I agree the default is better. But it still sounds quite dangerous if we make a mistake (don't realize we forgot to push a version of that package on our index). Looking forward to the pinned package index!

Thanks for the update.